providers/gitlab.js

```diff
@@ -0,0 +1,103 @@
+// gitlab.js
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/user/project/integrations/webhooks.md
+// ========
+module.exports = {
+    parse: function (req, discordPayload) {
+        var body = req.body;
+        var username = body.user_name;
+        var type = body.object_kind;
+        var ref = body.ref;
+        discordPayload.username = username + " Commit";
```

Comment:
As it seems that Jawnnypoo doesnt want to override the username and avatar of the webhook (he removed even my overrides for it) you should consider removing these lines.